pkg/api/volumes.go

```diff
@@ -366,6 +367,8 @@ func (client *Client) mapVolumeProcess(volumeName, initiatorName string, lun int
 		}
 	} else if metadata.ReturnCode == common.VolumeNotFoundErrorCode {
 		return status.Errorf(codes.NotFound, "volume %s not found", volumeName)
+	} else if metadata.ReturnCode == common.LUNOverlapErrorCode {
+		return status.Errorf(codes.AlreadyExists, "lun overlap for lun: %d", lun)
```

Comment:
One thought it to make sure that CSI Sanity still passes when codes.AlreadyExists is returned. Many csi sanity test cases rely on very specific return codes.

Reply:
I tested and confirmed this doesn't cause new csi sanity failures

Reply:
Thank you.